vms/platformvm/vm.go

```diff
@@ -52,6 +53,14 @@ import (
 	pvalidators "github.com/ava-labs/avalanchego/vms/platformvm/validators"
 )
 
+const (
+	txGossipBloomMaxItems             = 8 * 1024
+	txGossipBloomFalsePositiveRate    = 0.01
+	txGossipBloomMaxFalsePositiveRate = 0.05
+	txGossipThrottlingPeriod          = 10 * time.Second
+	txGossipThrottlingLimit           = 2
+)
```

Comment:
nit: should this config be in the gossip.go file?
Not sure if we keep it at vm level in other vms, in which case I'd rather be consistent and keep them here